hikari/internal/enums.py

```diff
@@ -122,8 +109,15 @@ def __call__(cls, value: typing.Any) -> typing.Any:
     def __getitem__(cls, name: str) -> typing.Any:
         return cls._name_to_member_map_[name]
 
+    def __contains__(cls, item: typing.Any) -> bool:
+        try:
+            cls._value_to_member_map_[item]
```

Comment:
how does this compare to just returning `item in cls._value_to_member_map_`?

Reply:
Same thing, just one call less, but will change if it makes it cleaner